src/scripting/ScriptContext.cpp

```diff
@@ -115,7 +115,7 @@ ScriptContext::ScriptContext(LuaSandbox& aLuaSandbox, const std::filesystem::pat
         return std::get<std::string>(acDescription);
     };
 
-    auto registerBinding = [this, &wrapHandler, &wrapDescription](
+    auto registerBinding = [this, &wrapHandler, &wrapDescription, &lockedState](
```

Comment:
you can just copy lockedState, it is simple pointer

Reply:
it is a shared pointer with a scoped lock, that is why I can't copy it. I can copy just a pointer or capture whole luaSandbox and lock inside. But I rather leave it as it is.

Reply:
ah you right had not realised that... state in sol3 is but this wraps it. k, leave it as is.